hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -1295,6 +1295,59 @@ bool MDMParser::powerState(void) {
     return state;
 }
 
+bool MDMParser::urcs(bool enable) {
+    if (enable) {
+        sendFormated("AT+UCIND=4095\r\n");
```

Comment:
Nit: might be a good idea to wrap these in `CHECK_TIMEOUT` on wake-up so that if the modem for whatever reason is unresponsive we'll go into an error state much quicker.